README.md

```diff
@@ -54,18 +54,18 @@ public class MainActivity extends SingleActivity {
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
-    setContentView(R.layout.main_activity);
+    setContentView(R.layout.activity_main);
   }
 
 }
 ```
 
-`main_activity.xml`:
+`activity_main.xml`:
```

Comment:
why changing to activity_main.xml ?
main_activity is better english no? @ryanmoelter

Reply:
`activity_main.xml` is the Android convention and Android Studio's default naming for the `MainActivity`'s layout xml. I personally like `main_activity` better but that's not what most people use.

Reply:
I personally prefer main_activity as well, but like Ryan said, that's not the Android convention. I renamed it in the example solely to make it more understandable, across the board :)

Reply:
Although, it is nice to be able to alphabetically see these by groupings in a big project:
activity_
dialog_
fragment_
layout_
list_item_
view_

etc.

Reply:
Ok, you two convinced me that it's easier to get started.

@coridrew : it's usually not optimal to group things in code based on type in my experience. :) A better grouping is usually to group by features: login_, profile_, settings_ etc. :)